PollBuddy-Server/frontend/src/App.js

```diff
@@ -148,6 +149,10 @@ export default class App extends React.Component {
               <Template updateTitle={this.updateTitle} />
             </Route>
 
+            <Route exact path="/polls/:pollID/history">
```

Comment:
Probably better as just /polls/history. Also can you move it up with the other polls routes, probably at line 111?

Reply:
Oops I didn't make this a review comment

Reply:
Moved and changed path and import in App.js